electrumutils.py

```diff
@@ -0,0 +1,212 @@
+import logging
+import binascii
+from utils import BITCOIND_CONFIG
+import time
+import sys
+import os
+import tempfile
+import traceback
+import asyncio
```

Comment:
I'm rather hesitant to mix concurrency paradigms, and the rest uses the `threading` module. Would you mind using that for the time being?

Reply:
Since both Electrum and ElectrumX rely on Asyncio, and we need to somehow schedule Tasks on the loop of Electrum, and somehow launch and shutdown ElectrumX (which also uses asyncio), I don't see how I can avoid importing Asyncio here.

1. One way of doing this without importing Asyncio in lightning-integration would entail having the call to run_coroutine_threadsafe in the Electrum codebase proper, and then simply calling that method from the lightning-integration test, but since it would be a method only used here, I don't see the value of having that in Electrum. By the way, we are already mixing concurrency paradigms in Electrum anyway, since e.g. Qt doesn't natively support the Asyncio loop. It is a necessary evil since not all software is written using the same concurrency paradigms. 
1. Another way to do it, would be to parse the log of some process, but I don't like this approach of string based RPC since the shared implementation language allows lightning-integration's Electrum parts to easily drill into Electrum structures. Why would we settle on some logging format and then parse it here?
1. Another way would be to make all of lightning-integration use asyncio (good idea but I don't have the time), or rolling back Electrum to use plain old threads, which makes no sense, Asyncio is working great for us.

Of all these alternatives, the most attractive one the first one, and I could do that, but I'd really prefer not to, because it would just be another layer of pointless indirection, since it'd be an abstraction that is exclusively used here.